drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/widget/table/utilities/ColumnUtilitiesTest.java

```diff
@@ -125,6 +132,148 @@ public void testCanAcceptOtherwiseWrongOperator() throws Exception {
         assertFalse(ColumnUtilities.canAcceptOtherwiseValues(column));
     }
 
+    @Test
+    public void testGetValueList() throws Exception {
+        testGetValueList("a,b,c", DataType.TYPE_STRING, 3, new String[] {"a","b","c"});
+    }
+
+    @Test
+    public void testGetValueListWhiteSpaces() throws Exception {
+        testGetValueList("a, b b b  ,c", DataType.TYPE_STRING, 3, new String[] {"a"," b b b  ","c"});
+    }
+
+    @Test
+    public void testGetValueListCharsAndNumber() throws Exception {
+        testGetValueList("a,1,b", DataType.TYPE_STRING, 3, new String[] {"a","1","b"});
+    }
+
+    @Test
+    public void testGetValueListSpecial() throws Exception {
+        testGetValueList("@,#,$", DataType.TYPE_STRING, 3, new String[] {"@","#","$"});
+    }
+
+    @Test
+    public void testGetValueListEmpty1() throws Exception {
+        testGetValueList(",a", DataType.TYPE_STRING, 2, new String[] {"","a"});
+    }
+
+    @Test
+    public void testGetValueListEmpty2() throws Exception {
+        testGetValueList("a,,b", DataType.TYPE_STRING, 3, new String[] {"a","","b"});
+    }
+
+    @Test
+    public void testGetValueListDateNotSet() throws Exception {
+        testGetValueList("a,10-10-2010,b", DataType.TYPE_DATE, 3, new String[] {"a","10-10-2010","b"});
```

Comment:
I find this one surprising! There's only one "date" in the value list; yet you're saying the expectation is all three are valid for a ```DataType.TYPE_DATE``` field.